velox/vector/FunctionVector.h

```diff
@@ -55,6 +52,29 @@ class Callable {
       const std::vector<VectorPtr>& args,
       const BufferPtr& elementToTopLevelRows,
       VectorPtr* result) = 0;
+
+  /// Applies 'this' to 'args' for 'rows' and returns the result in
```

Comment:
This comment is very long and repeats the comment for 'apply'. Duplicating comments has the same problem as duplicating code: hard to ensure two copies stay in sync. It is also hard to tell what's the difference between the two method. Let's remove the copy and change the comment to " Same as 'apply', but <describe the difference>".

Reply:
Fixed